### PR TITLE
[ Navigation Bar Sticky Style ] [ Footer Update ]

### PR DESCRIPTION
**Pull request by JXproject**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # ----
 # Site
 
-title: Hydra Template
-url: "https://tableuv.github.io"
-baseurl: "/"
+title: TableUV
+url:
+baseurl:
```
Comment (dj4park) [inline _config.yml:6]:
```suggestion
baseurl: "/"
```

File: _config.yml [hunk 2]
```diff
@@ -1,9 +1,11 @@
 # ----
 # Site
 
-title: Hydra Template
-url: "https://tableuv.github.io"
-baseurl: "/"
+title: TableUV
+url:
```
[the remark this answers is not in the document]
Comment (dj4park) [reply _config.yml:5]: When working locally, url should be set as empty. 
When pushing to github url should be set to "https://tableuv.github.io"

File: _config.yml (hunk 1 above)
Comment (dj4park) [reply _config.yml:6]: When working locally, baseurl should be set as empty.
When pushing to github baseurl should be set to "/"

**Review by dj4park — CHANGES_REQUESTED** [review]
Check the _config.yml

File: _config.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (JXproject) [reply _config.yml]: Changed

**Review by JXproject — COMMENTED** [review]
Changed upon request + updated footer

**Pull request merged**